### ISSUE TITLE
ER arrow heads are not working when selected on format pane

### ISSUE THREAD

**Issue by udithv** [jgraph/mxgraph#191]
the styles for ERone , ERmanytoone etc are not working when running the examples locally. When the arrow are selected and the startwith and endwith are toggled for other types it seems to be working but not for ER types.

**Comment by alderg** [comment]
Not sure what you mean. Could you describe the steps to reproduce the problem, and the expected and actual result, please?

**Comment by udithv** [comment]
I was running one of Javascript examples called the GraphEditor. I was using nodejs based simple-server to serve the files. When I was checking the GraphEditor format Pane that appears when you select an edge.I was trying out all the options in the dropdown menu for startwith and endwith for the arrow heads. It worked for every option except ER based arrows like one, manytoone etc.
The head of the arrowhead has to change to ERmany type but it doesn't . The result is the same for other options

So, I checked the Format.js file and there is call to a function called this.editorUi.menus.edgeStyleChange() one of its arguments is an array which looks like 
['ERone', 0]

Furthermore, I checked the Shapes.js file for ERone, ERmany etc..but failed to find any definitions for them. I would find only definitions upto circlePlus.

I tried the same with draw.io source files and ithe webapp based on the example worked fine. I even replaced the javascript files from draw.io source for webapp into this. Again everything seemed to work fine except the feature I am talking about which leads me to believe the problem is not from the js files..
